src/snowflake/connector/test_util.py

```diff
@@ -10,7 +10,9 @@
 
 from .compat import IS_LINUX
 
-RUNNING_ON_JENKINS = os.getenv("JENKINS_HOME") is not None
+
+JENKINS_HOME_DEFAULT_RETURN_VALUE= '$!$@46217d2d-1000-40b2-b4e5-9fdbab719c7c'
+RUNNING_ON_JENKINS = os.getenv("JENKINS_HOME", JENKINS_HOME_DEFAULT_RETURN_VALUE) is not JENKINS_HOME_DEFAULT_RETURN_VALUE
```

Comment:
You usually want to test equality of strings with `==`, or `!=` not with `is`.
```suggestion
RUNNING_ON_JENKINS = os.getenv("JENKINS_HOME", JENKINS_HOME_DEFAULT_RETURN_VALUE) != JENKINS_HOME_DEFAULT_RETURN_VALUE
```